### PR TITLE
[BD-46] feat: update DataTable font size to be small

### PR DESCRIPTION
**Pull request by monteri**
Font size equals to `$font-size-sm` for all elements in the table.

JIRA: [PAR-609](https://openedx.atlassian.net/browse/PAR-609)

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @monteri! I've created [BLENDED-1062](https://openedx.atlassian.net/browse/BLENDED-1062) to keep track of it in Jira.   More details are on the [BD-46](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3134685276/BD-46+Paragon+Blended+Support+and+Maintenance) project page.


When this pull request is ready, tag your edX technical lead.

File: src/DataTable/DataTable.scss [hunk 1]
```diff
@@ -22,6 +22,10 @@ $data-table-pagination-dropdown-min-width: 6rem !default;
     border-bottom-left-radius: $border-radius;
     border-bottom-right-radius: $border-radius;
   }
+
+  * {
+    font-size: $font-size-sm;
```
Comment (adamstankiewicz) [inline src/DataTable/DataTable.scss:27]: @monteri Looking over the Figma spec for `DataTable`, it seems we may want to be a bit more selective in what we set as `$font-size-sm`.

Not _all_ the content within DataTable is set as small, but its more focused around the content of the table that should have `$font-size-sm`, not everything. For example, the font sizes within the filters and actions stay normal size, but the "Showing 1 - 50..." both above and below the table contents and the table contents themselves should have `$font-size-sm`.

I'd probably just crosscheck with the [Figma spec](https://www.figma.com/file/eGmDp94FlqEr4iSqy1Uc1K/Paragon-Design-System?node-id=10469%3A37756) on which parts of `DataTable` should be small (i.e., by clicking on an element in Figma, you can see what should be small in the sidebar, see screenshot).

image

Should probably have been more specific in the acceptance criteria ;)

**Review by adamstankiewicz — CHANGES_REQUESTED** [review]

File: src/DataTable/DataTable.scss (hunk 1 above)
Comment (monteri) [reply src/DataTable/DataTable.scss]: Yes, thanks for clarification, I thought the ticket was meant to make whole text with the small text size :)
Now I placed `font-size: $font-size-sm;` only for the wrapper so it made the expected result as in the Figma

Comment (adamstankiewicz) [reply src/DataTable/DataTable.scss]: Awesome, thanks for the update!

**Review by adamstankiewicz — APPROVED** [review]

**Comment by edx-webhook** [comment]
@monteri 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/paragon&entry.867055334=https://github.com/edx/paragon/pull/1009&entry.1484655318=https://github.com/monteri&entry.752974735=2022-01-05+12:14&entry.1917517419=2022-01-06+17:26&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**